app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -406,8 +406,8 @@ def clean_up_extra_flavor_keys
       end
     end
 
-    def add_instance_disk(disks, size, location, name)
-      super(disks, size, location, name, "openstack")
+    def add_instance_disk(disks, size, location, name, controller_type)
+      super(disks, size, location, name, controller_type)
```

Comment:
the location is always unique per Vm, it goes usually as vda, vbd, vdc, ...so it works nicely with the old saving code